src/igtcloud/client/tools/upload_project.py

```diff
@@ -83,6 +87,80 @@ def upload_project(local_folder: str, project_name: str, institute_name: str = N
                             f"files_skipped: {len(files_skipped)}")
 
 
+def upload_annotation_files_private_method(institutes, local_folder, max_workers_files):
+    # Annotation file upload
+    logger.info("Annotation File Uploading...")
+    annotation_files = []
+    if os.path.isdir(local_folder):
+        for root, dirs, local_files in os.walk(local_folder):
+            local_path = root
+            annotation_files.extend(local_files)
+    else:
+        logger.error("Not a valid Directory")
+    initial_path = local_path.split("/")
```

Comment:
Forward slash will not work on Windows. This applies for other occurrences below.

Reply:
Aligned with both Windows and Linux path , other forward slash are s3 path related